Libraries/LibJS/Heap/Heap.cpp

```diff
@@ -157,6 +157,14 @@ void Heap::gather_conservative_roots(HashTable<Cell*>& roots)
         ASSERT_NOT_REACHED();
     }
     pthread_attr_destroy(&attr);
+#elif __MACH__
```

Comment:
Does this have anything to do with Mach? `__APPLE__` would be more appropriate here if you want to check for Darwin.

Reply:
Both `__APPLE__` and `__MACH__` are defined on my system. Tbh. I'm not sure why I chose the latter. I guess I'd need to submit another PR to fix this then? :D

Reply:
Yes, because macOS is a Darwin distribution, and Darwin uses Mach. But not anything that uses Mach is a version of Darwin :wink:

Yes, please submit a separate PR, and don't forget to cc me.